extra/1-currency-conversion.js

```diff
@@ -15,7 +31,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+	var newPrice = (price * 99) / 100;
+	var convertedBRL = (newPrice * 5.7).toFixed(2);
+	return parseFloat(convertedBRL);
```

Comment:
Maybe assign 5.7 to a variable called something like exchangeRate, just to make the code more understandable